README.md

```diff
@@ -242,6 +242,9 @@ git clone https://github.com/OpenVoiceOS/ovos-docker.git
 mkdir -p ~/ovos/{config,share,tmp} ~/hivemind/{config,share}
 chown ${USER}:${USER} -R ~/ovos ~/hivemind
 cd ~/ovos-docker/compose
+cp .env.example .env # customise if needed
+  # Or:
+cp .env-raspberrypi .env # customise if needed
```

Comment:
this really does depend on you. We can simply ``cp .env-raspberrypi .env`` and then edit all the commands in the README mentioning ``.env-raspberrypi``, or as you suggest, having ``cp .env-raspberrypi.example .env-raspberrypi`` and do the changes accordingly. For me it would be easier the first one as a user, the second is more instructive. Tell me and I will make the changes.

Reply:
We can keep `cp .env-raspberrypi .env` if you think it's easier but then `README.md` must be updated as well to reflect the change.